### DISCUSSION TITLE
Compare studies

### DISCUSSION THREAD

**Discussion by baburr** [optuna/optuna-dashboard#644] [Q&A]
Why do markers not appear in different colors when comparing two trainings on the history graph?
![image](https://github.com/optuna/optuna-dashboard/assets/44023210/9936e298-eb8d-401d-b966-79ab31aed53e)

**Comment by c-bata** [comment]
@keisuke-umezawa Could you respond to this question?

**Reply by keisuke-umezawa** [reply]
@baburr 
Do you have some snippets to reproduce this behavior?
